### ISSUE TITLE
Getting a response with private and protected properties

### ISSUE THREAD

**Issue by maronkoders** [geocoder-php/GeocoderLaravel#184]
### General Information
GeocoderLaravel Version: 
Laravel Version:  6.18.7
PHP Version: ^7.1.3
Operating System and Version:  Windows 10 Pro  (19041.867)

### Issue Description
[describe what is going wrong in plain english]

I have created my function that call the `$geocoder->geocode('Los Angeles, CA')->get()` . The problem is im getting private properties in the response body.Hence , i cannot work with it

### Steps to Replicate
[please provide a list of specific steps that will allow me to replicate this issue]

### Stack Trace
[if an error is occuring, provide the complete stack trace here]

**Comment by mikebronner** [comment]
Hi @maronkoders, if I understand you right, you aren't familiar with how to use the response you get back when making a query. This does indeed appear to be poorly documented by the Geocoder PHP documentation. However, the old documentation still shows the available methods: https://github.com/geocoder-php/Geocoder/blob/3.x/README.md#address--addresscollection

Hope this helps. :)

**Comment by maronkoders** [comment]
Let me check it out .Thanks
